.github/CONTRIBUTING.md

```diff
@@ -44,7 +43,6 @@ everything still works.
 ./test.sh
```

Comment:
Im not sure how to handle this portion. I updated the `test.sh` script to the appropriate directory now

Reply:
@kennedy We might want to adopt a build script like the one for the FOSS website: https://github.com/FOSSRIT/fossrit.github.io/blob/master/build.sh